src/main/java/teammates/common/util/SanitizationHelper.java

```diff
@@ -202,6 +202,7 @@ public static String desanitizeFromHtml(String sanitizedString) {
                               .replace("&quot;", "\"")
                               .replace("&#x2f;", "/")
                               .replace("&#39;", "'")
+                              .replace("&#34;", "\"")
```

Comment:
Tag @leeyimin here, I'm not too familiar with sanitization so I'm not sure if this is okay.